src/features/apps/marketplaceDeprecated/apiSlice.ts

```diff
@@ -0,0 +1,50 @@
+/********************************************************************************
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation
```

Comment:
File was introduced this year!
```suggestion
 * Copyright (c) 2025 Contributors to the Eclipse Foundation
```

Reply:
Changes done